daemon-tests/tests/happy_path.rs

```diff
@@ -261,13 +280,15 @@ async fn force_close_an_open_cfd() {
     // After CetTimelockExpired, we're only waiting for attestation
     expire!(cet timelock, order_id, maker, taker);
 
-    // Delivering the wrong attestation does not move state to `PendingCet`
-    deliver_event!(maker, taker, dummy_wrong_attestation());
+    // Decrypting the wrong attestation does not move state to `PendingCet`
+    decrypt_cet(&mut maker, &mut taker, order_id, dummy_wrong_attestation()).await;
```

Comment:
Any particular reason we are moving away from using a macro for this? The macro makes it a bit more concise because you don't need to `await` it and the mutable reference is not visible.

Reply:
I like plain functions because they work better with IDE tooling and are easier to read if they have to modified.

the trade off here is concise vs easier-to-work-on and I guess my preference is easier-to-work-on but happy to revert back to a macro

Reply:
Yes please keep the macro for consistency with the current code.